lib/pages/chat_blank/chat_blank.dart

```diff
@@ -95,18 +99,64 @@ class _ChatBlankRichText extends StatelessWidget {
   WidgetSpan _buildIconTitle(BuildContext context) {
     return WidgetSpan(
       alignment: PlaceholderAlignment.middle,
-      child: IconButton(
-        onPressed: () {
-          context.go('/rooms/newprivatechat');
-        },
-        icon: Icon(
-          Icons.mode_edit_outline_outlined,
-          size: ChatBlankStyle.iconSize,
-          color: Theme.of(context).primaryColor,
+      child: MenuAnchor(
+        menuChildren: [
+          MenuItemButton(
+            leadingIcon: const Icon(Icons.chat),
+            onPressed: goToNewPrivateChat,
+            child: Text(L10n.of(context)!.newDirectMessage),
+          ),
+          MenuItemButton(
+            leadingIcon: const Icon(Icons.group),
+            onPressed: goToNewGroupChat,
+            child: Text(L10n.of(context)!.newChat),
+          ),
+        ],
+        style: const MenuStyle(
+          alignment: Alignment.topLeft,
         ),
+        builder: (context, menuController, _) {
+          return InkWell(
+            onTap: () => menuController.open(),
+            child: Icon(
+              Icons.mode_edit_outline_outlined,
+              size: ChatBlankStyle.iconSize,
+              color: Theme.of(context).primaryColor,
+            ),
+          );
+        },
       ),
     );
   }
+
+  bool get goRouteAvailableInFirstColumn =>
+      FirstColumnInnerRoutes.instance.goRouteAvailableInFirstColumn();
+
+  BuildContext innerNavigatorContext() {
+    if (!goRouteAvailableInFirstColumn) {
+      return FirstColumnInnerRoutes
+          .innerNavigatorNotOneColumnKey.currentState!.context;
+    } else {
+      return FirstColumnInnerRoutes
+          .innerNavigatorOneColumnKey.currentState!.context;
+    }
+  }
+
+  void goToNewGroupChat() {
```

Comment:
you can't `GoToGroupChatMixin` file or adapt it ?

Reply:
I need to manage them separately at this screen, because it's special.

Reply:
what is the special here?

Reply:
Updated use `GoToGroupChatMixin `